src/core/data-abbr.js

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+// Module core/data-abbr
+// - Finds all elements with data-abbr attribute and processes them.
+import { showInlineWarning } from "./utils";
+export const name = "core/dfn-abbr";
+
+export function run() {
+  document.querySelectorAll("[data-abbr]").forEach(elem => {
+    const { localName } = elem;
+    switch (localName) {
+      case "dfn":
+        // @ts-ignore
```

Comment:
Why `@ts-ignore` is needed?

Reply:
Is it something related to ```HtmlElement``` and ```Element```? Breaking up the querySelectorAll and forEach loop could help.

Reply:
Ah yeah, let me do what you did originally @devanshbatra04.